entrypoint.R

```diff
@@ -1,30 +1,26 @@
 #!/usr/local/bin/Rscript
 
-dht::greeting(geomarker_name = 'st_census_tract',
-              version = '0.1.3',
-              description = 'links geocoded coordinates with date ranges to cooresponding census tracts from the appropriate vintage')
+dht::greeting()
+
+## load libraries without messages or warnings
+withr::with_message_sink("/dev/null", library(dplyr))
+withr::with_message_sink("/dev/null", library(tidyr))
+withr::with_message_sink("/dev/null", library(sf))
 
 old_warn <- getOption("warn")
 options(warn = -1)
```

Comment:
is this still necessary?

Reply:
This was to silence package load warnings, right? No, I don't think we need it.